src/main/resources/copper_golem.mixin.json

```diff
@@ -1,13 +1,12 @@
 {
-    "required": true,
-    "minVersion": "0.7",
-    "package": "com.sollace.coppergolem.mixin",
-    "refmap": "copper_golem.mixin.refmap.json",
-    "compatibilityLevel": "JAVA_16",
-    "mixins": [
-    	"MixinAbstractButtonBlock",
-    	"MixinMobEntity"
-    ],
-    "client": [
-    ]
+  "required": true,
+  "minVersion": "0.7",
+  "package": "com.sollace.coppergolem.mixin",
+  "refmap": "copper_golem.mixin.refmap.json",
+  "compatibilityLevel": "JAVA_16",
+  "mixins": [
+    "MixinAbstractButtonBlock",
+    "MixinMobEntity"
+  ],
+  "client": []
```

Comment:
`jq` likes to compact empty arrays. Let me know if you prefer the two-line version.

Reply:
Single-line empty arrays is preferrable. Thanks!